src/components/home/Filter.js

```diff
@@ -0,0 +1,21 @@
+const flterStyle = {
+  transform: 'translateY(-1.5rem)',
+  marginLeft: '1rem',
+};
+
+const flterInput = {
+  color: 'black',
+  padding: '0.4rem',
+  border: 'none',
+  outline: 'none',
+  width: '30%',
+};
+
+// eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Kindly remove the eslint disable line and instead fix the issue in the code.